docroot/modules/features/resource/config/optional/views.view.mirador_view.yml

```diff
@@ -0,0 +1,267 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.mirador
+    - field.storage.node.field_mi
+    - node.type.resource
+  module:
+    - node
+    - user
+id: mirador_view
+label: 'Mirador View'
+module: views
+description: ''
+tag: ''
+base_table: node_field_data
+base_field: nid
+core: 8.x
+display:
+  default:
+    display_plugin: default
+    id: default
+    display_title: Master
+    position: 0
+    display_options:
+      access:
+        type: perm
+        options:
+          perm: 'access content'
```

Comment:
Annotations are tied up with authenticated users only. Can we set the appropriate permission here. Set it up on the basis of roles.